go/libkb/passphrase_helper.go

```diff
@@ -124,7 +124,7 @@ func DefaultPassphraseArg() keybase1.GUIEntryArg {
 				Label:        "Show typing",
 			},
 			StoreSecret: keybase1.Feature{
-				Allow:        HasSecretStore(),
+				Allow:        G.SecretStore != nil,
```

Comment:
I think you'll have to pass GlobalContext into this function to avoid using capital-G G.